tests/test_builtin_adapters.py

```diff
@@ -68,6 +68,14 @@ def test_disk_vs_string(self):
             with open(temp_file, 'r') as f:
                 on_disk = f.read()
 
+            self.maxDiff = None
+
+            with open("/var/tmp/in_memory.otio", "w") as fo:
```

Comment:
Should this use the `tempfile` library? Also I'm struggling to understand why you write into a file without reading it later in the test.

Reply:
Diagnostic code that was left in.  Removed!